chapters/classes.tex

```diff
@@ -71,7 +71,7 @@ \section{Double Declaration not Allowed}\label{double-declaration-not-allowed}
 \end{lstlisting}
 \end{example}
 
-\section{Declaration Order and Usage before Declaration}\label{declaration-order-and-usage-before-declaration}
+\section{Declaration Order - Usage before Declaration}\label{declaration-order-usage-before-declaration}\label{declaration-order-and-usage-before-declaration}
```

Comment:
En-dash:
```suggestion
\section{Declaration Order -- Usage before Declaration}\label{declaration-order-usage-before-declaration}\label{declaration-order-and-usage-before-declaration}
```

Reply:
How about even dropping the part after the dash entirely?
```suggestion
\section{Declaration Order}\label{declaration-order}\label{declaration-order-and-usage-before-declaration}
```

Reply:
Yes.